node-require/divide.js

```diff
@@ -0,0 +1,3 @@
+const divide = (x, y) => x / y;
+
+module.exports.divide = divide;
```

Comment:
If you assign your function to the `exports` property, you can eliminate the need to do `divideFile.divide`

```js
module.exports = divide;
```

```js
const divide = require('./divide');
```